.gitignore

```diff
@@ -1,32 +1,29 @@
 # Ignore creds file
-*/**/auth/auth.yml
```

Comment:
Fixed .gitignore so it doesn't require a subdirectory for the files, especially for config.yml which is located in the root directory and not inside a subdirectory.

See: 
https://stackoverflow.com/questions/681262/difference-in-the-paths-in-gitignore-file